pkgs/development/python-modules/pyside/tools.nix

```diff
@@ -13,7 +13,9 @@ buildPythonPackage rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ cmake pyside qt4 pysideShiboken ];
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ pyside qt4 pysideShiboken ];
```

Comment:
I don't get why `pyside` and `pysideShiboken` were not put in `propagatedBuildInputs` in the first place since they are Python packages. It would be nice if you could change that.

Reply:
I can do that.  Can you refer me to the place in the documentation that explains why this is the correct thing to do.

Reply:
See e.g. https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#what-inputs-do-setup_requires-install_requires-and-tests_require-map-to and "Handling dependencies" under https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/download/1/nixpkgs/manual.html#developing-with-python.